apps/core/tests/test_models.py

```diff
@@ -369,14 +380,19 @@ def setUp(self):
             password='toto,1234!'
         )
         self.analysis = models.Analysis.objects.create(
-            secondary_data='/fake/path/secondary_data',
             pixeler=pixeler,
+            completed_at=datetime.date(1980, 10, 14),
+        )
+        self.analysis.secondary_data.name = (
+            factories.SECONDARY_DATA_DEFAULT_PATH
         )
+        self.analysis.save()
```

Comment:
Why saving here?

Reply:
To ensure that `analysis.secondary_data` file name is up-to-date in the database, not only in the model instance.